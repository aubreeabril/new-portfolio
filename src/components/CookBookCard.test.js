import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import CookbookCard from "./CookBookCard";

describe("CookBook Card", () => {
  it("renders the cookbook card", () => {
    const wrapper = shallow(<CookbookCard />);
    expect(wrapper.find(".cook-book").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    const section = renderer.create(<CookbookCard />).toJSON();
    expect(section).toMatchSnapshot();
  });
});
