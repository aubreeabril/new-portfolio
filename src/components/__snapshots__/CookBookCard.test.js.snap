// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookBook Card matches its snapshot 1`] = `
<div
  className="project-card cook-book card"
>
  <div
    className="card-image"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowfullscreen={true}
      frameborder="0"
      src="https://www.youtube.com/embed/gSKpxNr7-_s"
      title="Cookbook demo"
      width="100%"
    />
  </div>
  <h5
    className="card-title"
    style={
      Object {
        "padding": "10px",
      }
    }
  >
    Flatiron Cookbook
  </h5>
  <p
    className="project-text card-text"
  >
    An application for finding and saving recipes. Built with a Rails API backend and React frontend, styled with Semantic-React.
  </p>
</div>
`;
